react/features/notifications/components/native/Notification.tsx

```diff
@@ -0,0 +1,302 @@
+/* eslint-disable lines-around-comment */
+
+import React from 'react';
+import { WithTranslation } from 'react-i18next';
+import { Text, View } from 'react-native';
+
+import { IReduxState } from '../../../app/types';
+// @ts-ignore
+import { Avatar } from '../../../base/avatar';
+import { translate } from '../../../base/i18n/functions';
+import {
+    Icon,
+    IconCloseLarge,
+    IconInfoCircle,
+    IconUsers,
+    IconWarning
+    // @ts-ignore
+} from '../../../base/icons';
+import { getRemoteParticipants } from '../../../base/participants/functions';
+import { connect } from '../../../base/redux/functions';
+import { colors } from '../../../base/ui/Tokens';
+import BaseTheme from '../../../base/ui/components/BaseTheme.native';
+import Button from '../../../base/ui/components/native/Button';
+import IconButton from '../../../base/ui/components/native/IconButton';
+import { BUTTON_TYPES } from '../../../base/ui/constants.native';
+import { replaceNonUnicodeEmojis } from '../../../chat/functions';
+import { getKnockingParticipants } from '../../../lobby/functions';
+import { NOTIFICATION_ICON } from '../../constants';
+import AbstractNotification, {
+    type Props as AbstractNotificationProps
+    // @ts-ignore
+} from '../AbstractNotification';
+
+// @ts-ignore
+import styles from './styles';
+
+
+/**
+ * Default value for the maxLines prop.
+ *
+ * @type {number}
+ */
+const DEFAULT_MAX_LINES = 2;
+
+/**
+ * Secondary colors for notification icons.
+ *
+ * @type {{error, info, normal, success, warning}}
+ */
+
+const ICON_COLOR = {
+    error: colors.error06,
+    normal: colors.primary06,
+    success: colors.success05,
+    warning: colors.warning05
+};
+
+
+type Props = AbstractNotificationProps & WithTranslation & {
+    _participants: ArrayLike<any>;
```

Comment:
This is really weird. Why does a notification need to have a reference to an array of participants?

Reply:
I use this for the avatar to show. We show it only for one participant lobby notification.

Reply:
Mentioned below, let's drop the avatar please.